src/Core/Commander/Providers/Fetcher.js

```diff
@@ -0,0 +1,41 @@
+import { TextureLoader } from 'three';
```

Comment:
> (Especially for loading)

What do you mean by that? Importing it one or n times won't change the loading time, it's just a scope thing AFAIK (hopefully so, otherwise `import` syntax would have to be avoided at all cost!)

>  The Fetcher can return an image and not a texture.

That would mean nearly exactly duplicating what Three.js already does (see [TextureLoader](https://github.com/mrdoob/three.js/blob/6c7f000734f8579da37fb39e5c2e9e5e2dfb14f8/src/loaders/TextureLoader.js)). So a bigger code (so slower to load), that we need to maintain.

I feel like I'm missing the point: why importing and using an utility function is "contaminating the code"? And why do we want to limit the "expansion of three.js in itowns"? (I don't know what you mean by "expansion" and "contamination" btw). Do we plan on being agnostic regarding the WebGL lib?

Or do you mean you feel this function should be elsewhere?